storage/client/README.md

```diff
@@ -0,0 +1,358 @@
+# Azure Storage Queue client library for Java
+Azure Queue storage is a service for storing large numbers of messages that can be accessed from anywhere in the world via authenticated calls using HTTP or HTTPS. 
+A single queue message can be up to 64 KB in size, and a queue can contain millions of messages, up to the total capacity limit of a storage account.
+
+[Source code][source_code] | [Package (Maven)][package] | [API reference documentation][api_documentation] | [Product documentation][storage_docs]
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk] with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Create Strorage Account][storage_account]
+
+### Adding the package to your product
+
+```xml
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-storage</artifactId>
+  <version>12.0.0</version>
+</dependency>
+```
+
+### Create a Storage Account
+To create a Storage Account you can use the Azure Portal or [Azure CLI][azure_cli].
+
+```Powershell
+az group create \
+    --name storage-resource-group \
+    --location westus
+```
+
+### Authenticate the client
+
+In order to interact with the Storage service (Blob, Queue, Message, MessageId, File) you'll need to create an instance of the Service Client class. 
+To make this possible you'll need the Account SAS (shared access signature) string of Storage account. Learn more at [SAS Token][sas_token]
+
+#### Get Credentials
+
+- SAS Token
+ 
+a. Use the [Azure CLI][azure_cli] snippet below to get the SAS token from the Storage account.
+
+```Powershell
+az storage queue generate-sas
+    --name {queue name}
+    --expiry {date/time to expire SAS token}
+    --permission {permission to grant}
+    --connection-string {connection string of the storage account}
+```
+
+```Powershell
+#usage example
+az storage queue generate-sas
+    --name javasdksas
+    --expiry 2019-06-05
+    --permission rpau
+    --connection-string DefaultEndpointsProtocol=https;AccountName=azstoragesdkaccount;AccountKey=PhOFTkuqB09cmsy0aMgd+cJfp75xU3dgjRMKrWxHHx4pFvuFIMOSK/fhzGci6zxFGk4CkmgoXFO3s91S9m+ICw==;EndpointSuffix=core.windows.net
+```
+b. Alternatively, get the Account SAS Token from the Azure Portal.
+```
+Go to your storage account -> Shared access signature -> Click on Generate SAS and connection string (after setup)
+```
+
+- Shared Key Credential
+a. Use account name and account key. Account name is your storage account name.
```

Comment:
Need another new line here, Shared Key Credential and the Use account name... are on the same line

Reply:
Done.